warehouse/models/rt_views/_rt_views.yml

```diff
@@ -1,6 +1,45 @@
 version: 2
 
 sources:
+  - name: gtfs_rt_external_tables
+    description: New hive-partitioned external tables reading from GCS.
+    database: cal-itp-data-infra
+    schema: external_gtfs_rt
+    tables:
+      - name: service_alerts
+      - name: service_alerts_outcomes
+        loaded_at_field: cast(dt as timestamp)
+        freshness:
+          warn_after: { count: 1, period: day }
+      - name: service_alerts_validations
+      - name: service_alerts_validations_outcomes
+        loaded_at_field: cast(dt as timestamp)
+        freshness:
+          warn_after: { count: 1, period: day }
+      - name: trip_updates
+      - name: trip_updates_outcomes
+        loaded_at_field: cast(dt as timestamp)
+        freshness:
+          warn_after: { count: 1, period: day }
+      - name: trip_updates_validations
+      - name: trip_updates_validations_outcomes
+        loaded_at_field: cast(dt as timestamp)
+        freshness:
+          warn_after: { count: 1, period: day }
+      - name: vehicle_positions
+      - name: vehicle_positions_outcomes
+        loaded_at_field: cast(dt as timestamp)
+        freshness:
+          warn_after: { count: 1, period: day }
+      - name: vehicle_positions_validations
+      - name: vehicle_positions_validations_outcomes
+        loaded_at_field: cast(dt as timestamp)
+        freshness:
+          warn_after: { count: 1, period: day }
+
+
+
+  # Note: this source is deprecated by the above
```

Comment:
Do we have a plan for how to actually deprecate these? I guess that's part of #1468?

Reply:
Yes, exactly.